src/index.js

```diff
@@ -1,37 +1,76 @@
 import './style.css';
+import {
+  CHECK,
+  UNCHECK,
+  LINE_THROUGH,
+  TRASH_ICON,
+  LIST,
+  completeToDo,
+} from './completeToDo';
 
-const button = document.querySelector('.fa-reply');
-button.addEventListener('click', () => {
-  console.log('Lets drink coffee');
+// select the elements
+const list = document.getElementById('todo-list');
+const input = document.getElementById('input');
+const ELIPSIS = 'fa-ellipsis-v';
+
+// Variables
+let index;
+
+function addToDo(description, index, complete) {
+  const DONE = complete ? CHECK : UNCHECK;
+  const LINE = complete ? LINE_THROUGH : '';
+  const TRASH = complete ? TRASH_ICON : ELIPSIS;
+
+  const item = `<li class="item">
+        <i class="far ${DONE} co" job="complete" id="${index}"></i>
+        <p class="text ${LINE}">${description}</p>
+        <i class="trash fas ${TRASH} " job="delete" id="${index}"></i>
+    </li>
+    `;
+  const position = 'beforeend';
+  list.insertAdjacentHTML(position, item);
+}
+
+// load items to the user's interface
+function loadList(array) {
+  array.forEach((item) => {
+    addToDo(item.description, item.index, item.complete);
+  });
+}
+
+// get item from the localStorage
+const data = localStorage.getItem('TODO');
+if (data) {
+  LIST = JSON.parse(data);
+  index = LIST.length;
+  loadList(LIST);
+} else {
+  LIST = [];
+  index = 0;
+}
+
+document.addEventListener('keyup', (event) => {
+  if (event.keyCode === 13) {
+    const description = input.value;
+    if (description) {
+      addToDo(description, index, false);
+      LIST.push({
+        description,
+        index,
+        complete: false,
+      });
+      localStorage.setItem('TODO', JSON.stringify(LIST));
+      index += 1;
+    }
+    input.value = '';
+  }
 });
-const listToDo = [
-  {
-    completed: true,
-    description: 'Drink a coffee',
-    index: 0,
-  },
-  {
-    completed: false,
-    description: 'Do JavaScript project',
-    index: 1,
-  },
-];
-const list = document.querySelector('#todo-list');
-listToDo.forEach((todoItem) => {
-  const unorderedList = document.createElement('li');
-
-  const checkButton = document.createElement('div');
-  checkButton.id = 'checkList';
-  checkButton.innerHTML = '<i class="far fa-square"></i>';
-  unorderedList.append(checkButton);
-
-  const newToDo = document.createElement('p');
-  newToDo.innerText = todoItem.description;
-  unorderedList.append(newToDo);
-
-  const moveButton = document.createElement('div');
-  moveButton.id = 'moveList';
-  moveButton.innerHTML = '<i class="fas fa-ellipsis-v"></i>';
-  unorderedList.append(moveButton);
-  list.append(unorderedList);
+
+list.addEventListener('click', (event) => {
+  const element = event.target;
+  const elementJob = element.attributes.job.value;
+  if (elementJob === 'complete') {
+    completeToDo(element);
+  }
+  localStorage.setItem('TODO', JSON.stringify(LIST));
```

Comment:
**Changes Required**
- [ ] _Store the updated array of items in local storage_, so the user gets the correct list values after the page reloads. Currently, no localStorage is being created. Let us try to fix this please.